assets/bingo/lang/en_us.json

```diff
@@ -1,5 +1,6 @@
 {
 	"%1074992263$s%1$s": "%2$s",
+	"%1074992263$s%3$s%1$s": "%2$s",
 	"bingo.technical.detect_multiplayer": "DoNotTranslateThis",
```

Comment:
Didn't we want an index for this one too (i.e. `DoNotTranslateThis3`), so in case someone applied and overwrote the rp in singleplayer for some reason, they won't pass the automatic check?
We just don't need two separate strings like we do for the `%$2s` translations.

Reply:
Oh right, I misunderstood that part, as if you want to leave it as it is [2]
> It should be sufficient to keep `one DoNotTranslateThis string[1]` though, as it's `only used for checking whether the player should be teleported immediately[2]`, so having the logic in the chat message should be sufficient.

[1] and understood _one throughout versions_ 😅

Reply:
Done in https://github.com/NeunEinser/bingo/pull/111/commits/9b72b84c6a8446d77b32271cb459c7f4b144d2e6